darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -150,7 +150,13 @@ def training_step(self, train_batch, batch_idx) -> torch.Tensor:
             -1
         ]  # By convention target is always the last element returned by datasets
         loss = self._compute_loss(output, target)
-        self.log("train_loss", loss, batch_size=train_batch[0].shape[0], prog_bar=True)
+        self.log(
+            "train_loss",
+            loss,
+            batch_size=train_batch[0].shape[0],
+            prog_bar=True,
+            sync_dist=True,
```

Comment:
The PTL [doc](https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html) says `Use with care as this may lead to a significant communication overhead.`.
Do we have any idea if/when this could cause issues?

Reply:
So far in practical testing on 8 GPU-s I noticed no adverse effects. Thus said, it depends also on the distribution strategy also. I used the default `ddp_spawn`, as mentioned.